torch_ipex/csrc/cpu/ShadeDataContext.h

```diff
@@ -101,9 +101,6 @@ struct ShadeDataContext {
         TORCH_INTERNAL_ASSERT_DEBUG_ONLY(check_tensor_own_whole_storage(tensor));
         return true;
       } else {
-        // The dnnl tensor shares some data with raw tensor.
-        TORCH_INTERNAL_ASSERT_DEBUG_ONLY(shade_data_context->dil_tensor->is_public_format());
```

Comment:
Did you hit the assert?
1. If the ideep tensor is block-format, then the data of aten tensor should be nullptr.
2. If the ideep tensor is plain-format, then the data of aten tensor should be as same as the data handle of the ideep tensor.

Reply:
it was caused by previous incorrect setting of aten tensor data. Once I fix the shade_data_ptr, the assertion failure was gone.